lib/rules/no-await-sync-events.ts

```diff
@@ -25,9 +39,9 @@ export default createTestingLibraryRule<Options, MessageIds>({
 		docs: {
 			description: 'Disallow unnecessary `await` for sync events',
 			recommendedConfig: {
-				dom: 'error',
-				angular: 'error',
-				react: 'error',
+				dom: ['error', { eventModules: ['fire-event'] }],
+				angular: ['error', { eventModules: ['fire-event'] }],
+				react: ['error', { eventModules: ['fire-event'] }],
```

Comment:
This way we make sure we never have a typo and if we would remove `FIRE_EVENT_OPTION`, we'll get a TS error here

```suggestion
				dom: ['error', { eventModules: [FIRE_EVENT_OPTION] }],
				angular: ['error', { eventModules: [FIRE_EVENT_OPTION] }],
				react: ['error', { eventModules: [FIRE_EVENT_OPTION] }],
```